source/features/html-preview-link.tsx

```diff
@@ -15,7 +15,7 @@ function add(rawButton: HTMLAnchorElement): void {
 		.parentElement! // `BtnGroup`
 		.prepend(
 			<a
-				className="btn btn-sm BtnGroup-item rgh-html-preview-link"
+				className="btn btn-sm BtnGroup-item"
```

Comment:
Added in https://github.com/refined-github/refined-github/commit/86c0a731a3bb123b7a48fb0e2e21f4192df4fb46#diff-5a0d6407fb78a28cac196b6bf9d7f6e9eccaa02e5098d74df02eb70991c5ee5c for deduplicate, no longer used